index.html

```diff
@@ -0,0 +1,97 @@
+                  <p>
+                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus non aliquam est. Nulla luctus
+                      ligula quis pretium mollis. Duis placerat congue quam, ac tincidunt ligula iaculis quis. Ut
+                      egestas
+                      arcu sed nunc faucibus maximus.
+                      <p></p>
+                      Suspendisse eu ligula et ex lacinia malesuada. In eget nibh est. Morbi orci lectus, blandit eu
+                      imperdiet volutpat, sagittis quis leo. Etiam suscipit bibendum ex eget rhoncus. Sed tempor, quam
+                      vel
+                      sagittis interdum, nunc ligula accumsan ligula, non viverra massa risus et dui. Suspendisse
+                      ornare
+                      gravida arcu, in consequat justo mollis in. Nullam convallis dolor in aliquet tempus. Aliquam
+                      accumsan neque turpis, facilisis pretium leo vestibulum vitae. Donec libero nisl, elementum eu
+                      consequat vel, vestibulum id lacus. Integer porta est vel mauris consectetur, a vestibulum metus
+                      ultricies.
+                  </p>
+              </div>
+          </div>
+      </div>
+  </section>
+  <section id="pg2">
+      <div class="sect">
+          <div class="center">
+              <img src="pictures/head.png" alt="image">
+              <h1>iPhone Repair & Service</h1>
+          </div>
+          <div class="flex-container">
+              <div class="flex-item-left">
+                  <img src="pictures/screw.png" alt="screwdriver" width="60px">
+                  <h5>Schedule a visit</h5>
+                  <p>Make an appointment at an Apple Authorized Service Provider or Apple Store</p>
+              </div>
+              <div class="flex-item-right">
+                  <img src="pictures/box.png" alt="box">
+                  <h5>Send in for repair</h5>
+                  <p>Arrange to mail your product directly to Apple</p>
+              </div>
+          </div>
+      </div>
+  </section>
+  <section id="pg3">
+      <div class="sect">
+          <form id="form" action="https://www.freecodecamp.com/email-submit" class="center" style="width:450px">
+              <h1>Contact us</h4>
+                  <input id="username" name="username" type="text" placeholder="Insert your name">
+                  <input id="email" name="email" type="email" placeholder="Insert email">
+                  <textarea name="message" id="messsage" cols="30" rows="10"></textarea>
+                  <input id="submit" type="submit">
+          </form>
+      </div>
+  </section>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
The comments it using to guide you, so don't forget to remove all comments before you push your project

Reply:
- [ ] Awesome! Kindly ensure that your comment comes as a suggestion and as an order.